### ISSUE TITLE
No issue, but what does it do?

### ISSUE THREAD

**Issue by RobertoCravallo** [DJF3/Awtrix-Postman-collection-for-Ulanzi-clock#1]
Hey,

I am pretty new to this AWTRIX thing and I have no idea what this does or how to use it.

Please enlighten me / us non geeks...  ;-)

**Comment by DJF3** [comment]
Awtrix is alternative firmware for the Ulanzi clock. With this firmware you have full control over the clock, what is displayed etc. It allows the clock to be connected to Home automation systems like "Home Assistant". This results into use cases like: 
- Show the consumption of water/power/gas every hour
- Show when the doorbell rings
- Show an indicator when I have to put the trash out 
- and more.

You could also search YouTube to see how to convert a clock and see different usecases.

**Comment by RobertoCravallo** [comment]
Yes, I know and I flashed my clock. I still don't understand what the Postman does or how to use it....
